k8s/apps/deployments.go

```diff
@@ -21,7 +21,7 @@ type DeploymentOps interface {
 	// GetDeployment returns a deployment for the give name and namespace
 	GetDeployment(name, namespace string) (*appsv1.Deployment, error)
```

Comment:
Should we allow use of Get/update/delete options for all of sched-ops api as well

Reply:
Sure, we can add options to all APIs. 
The clients can update their code when they vendor the new changes.

Reply:
thanks,I will create separate PR